### DISCUSSION TITLE
.eslintrc appears ignored?

### DISCUSSION THREAD

**Discussion by kaihendry** [eslint/eslint#15340] [Q&A]
I expected "console.log" to be intended with a tab not two spaces. What am I missing please?

```
(ins)[hendry@t14s aa]$ cat index.js
function hello() {
  console.log("hello world");
}

hello();
(ins)[hendry@t14s aa]$ cat package.json
{
  "name": "aa",
  "version": "1.0.0",
  "scripts": {
    "lint": "eslint --fix; prettier --write ./"
  },
  "description": "",
  "main": "index.js",
  "keywords": [],
  "author": "",
  "license": "ISC"
}
(ins)[hendry@t14s aa]$ cat .eslintrc
{
  "env": {
    "node": true,
    "es6": true,
    "jest": true
  },
  "extends": ["eslint:recommended", "plugin:prettier/recommended"],
  "parserOptions": {
    "ecmaVersion": 2020,
    "ecmaFeatures": {
      "experimentalObjectRestSpread": true
    }
  },
  "rules": {
    "no-useless-catch": 0,
    "indent": ["error", "tab"]
  }
}
```

**Comment by aladdin-add** [comment]
what if renaming to `.eslintrc.json`? `.eslintrc` is not a valid config, as seen in the docs: https://eslint.org/docs/user-guide/configuring/configuration-files#configuration-file-formats

**Reply by kaihendry** [reply]
Thanks, though renaming the file and changing indent options seems to make no impact. Perhaps the `prettier --write` is overwriting any changes?

I opened https://github.com/eslint/eslint/issues/15344 since it's not clear to me how to integrate eslint with prettier.

**Comment by aladdin-add** [comment]
>     "lint": "eslint --fix; prettier --write ./"

why using `;`here?
as I know, this should work: `eslint --fix ./ && prettier --write ./`
